### PR TITLE
PLA-20459: Add rule information for the remediation job

### PR DESCRIPTION
**Pull request by mzkhan**

### CODE REVIEW

**Review by vikramsinghvirdi — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -43,13 +43,8 @@ The worker executes jobs in a fashion similar to running `python ./s3-remove-pub
 The finding payload is in the form:
 ```$json
 {
-  "cloudAccount": {
```
Comment (bhanu13) [inline README.md:46]: Good that we updated the payload as well.

**Review by bhanu13 — APPROVED** [review]

File: remediation_worker/jobs/s3_remove_public_access/README.md [hunk 2]
```diff
@@ -2,6 +2,22 @@
...
+
+##### Rule ID:
+* 5c8c26507a550e1fb6560c57
+* 5c8c26517a550e1fb6560c59
+* 5c8c26537a550e1fb6560c5a
+* 5c8c26537a550e1fb6560c5b
+* 5c8c26547a550e1fb6560c5c
+
+##### Rule Name:
+* An S3 bucket is configured so that all users have unrestricted permisions
```
Comment (bhanu13) [inline remediation_worker/jobs/s3_remove_public_access/README.md:15]: small typo

Comment (mzkhan) [reply remediation_worker/jobs/s3_remove_public_access/README.md]: fixed

**Review by bhanu13 — APPROVED** [review]

**Review by vikramsinghvirdi — APPROVED** [review]

**Pull request merged**